src/DynamoCoreWpf/HelixRenderPackage.cs

```diff
@@ -0,0 +1,577 @@
+        {
+            get
+            {
+                var hasData = points.Positions.Count > 0 || 
+                    lines.Positions.Count > 0 ||
+                    mesh.Positions.Count > 0;
+                return hasData;
+            }
+        }
+
+        /// <summary>
+        /// A flag indicating whether the render package is displaying labels
+        /// </summary>
+        public bool DisplayLabels { get; set; }
+
+        /// <summary>
+        /// A flag indicating whether the render package requires 
+        /// per vertex coloration.
+        /// </summary>
+        public bool RequiresPerVertexColoration { get; set; }
+
+        /// <summary>
+        /// The number of point vertices in the package.
+        /// </summary>
+        public int PointVertexCount
+        {
+            get { return points.Positions.Count; }
+        }
+
+        /// <summary>
+        /// The number of line vertices in the package.
+        /// </summary>
+        public int LineVertexCount
+        {
+            get { return lines.Positions.Count; }
+        }
+
+        /// <summary>
+        /// The number of mesh vertices in the package.
+        /// </summary>
+        public int MeshVertexCount
+        {
+            get { return mesh.Positions.Count; }
+        }
+
+        public IEnumerable<int> LineStripVertexCounts
+        {
+            get
+            {
+                return lineStripVertexCounts;
+            }
+        }
+
+        public IEnumerable<double> LineStripVertexBuffer
+        {
+            get { return lines.Positions.ToDoubleArray(); }
+        }
+
+        public IEnumerable<byte> LineStripVertexColorBuffer
+        {
+            get { return lines.Colors.ToByteArray(); }
+        }
+
+        public IEnumerable<int> LineStripIndexBuffer
+        {
+            get { return lines.Indices.ToArray(); }
+        }
+
+        public IEnumerable<double> MeshVertexBuffer
+        {
+            get { return mesh.Positions.ToDoubleArray(); }
+        }
+
+        public IEnumerable<byte> MeshVertexColorBuffer
+        {
+            get { return mesh.Colors.ToByteArray(); }
+        }
+
+        public IEnumerable<int> MeshIndexBuffer
+        {
+            get { return mesh.Indices.ToArray(); }
+        }
+
+        public IEnumerable<double> MeshNormalBuffer
+        {
+            get { return mesh.Normals.ToDoubleArray(); }
+        }
+
+        public IEnumerable<double> MeshTextureCoordinateBuffer
+        {
+            get { return mesh.TextureCoordinates.ToDoubleArray(); }
+        }
+
+        public IEnumerable<double> PointVertexBuffer
+        {
+            get { return points.Positions.ToDoubleArray(); }
+        }
+
+        public IEnumerable<byte> PointVertexColorBuffer
+        {
+            get { return points.Colors.ToByteArray(); }
+        }
+
+        public IEnumerable<int> PointIndexBuffer
+        {
+            get { return points.Indices.ToArray(); }
+        }
```

Comment:
Isn't it erroneous to call these properties `SomethingBuffer`?  Buffers have a very specific meaning that is not enforced by your type.